src/components/custom/other-platforms.js

```diff
@@ -336,7 +336,7 @@ export const NavMarket = ({ onClick, setCurrentPage }) => {
     useEffect(() => {
         if (Object.values(links).includes(window.location.pathname + window.location.hash))
             setCurrentPage('markets')
-    }, [window?.location.hash])
+    }, [isBrowser() ? window.location.hash : null])
```

Comment:
There is a function in `utility` that has the same implementation called `getLocationHash` it's better to use it here

Reply:
I just noticed this. Will change, thanks 😄